Makd.mak

```diff
@@ -487,7 +487,8 @@ clean:
 
 # Phony target to build all packages
 .PHONY: pkg
-pkg: $(patsubst $(PKG)/%.pkg,$O/pkg-%.stamp,$(wildcard $(PKG)/*.pkg))
+PKGS ?= $(wildcard $(PKG)/*.pkg)
+pkg: $(patsubst $(PKG)/%.pkg,$O/pkg-%.stamp,$(PKGS))
```

Comment:
Mmm, I see that you are still leaving the `$(patsubst )` in the rule. I think then it might be better to do the following to have both convenience and full flexibility:

* Use a `PKG_FILES` variable to simply specify package files to build, but define it *before* `Build.mak` is included (`PKG_FILES := $(wildcard $(PKG)/*.pkg)` so it can be overridden and simply `$(filter-out )`ed).
* Use a `pkg` variable to accumulate dependencies for the `pkg` target (like real targets)
* Assign the converted `PKG_FILES` as `pkg` targets `pkg += $(patsubst $(PKG)/%.pkg,$O/pkg-%.stamp,$(PKG_FILES))`

This way you can define the `PKG_FILES` from scratch, or you can `$(filter-out )` the defaults, and yet you can have any other crazy method to build packages and add/remove dependencies to `make pkg`.